### DISCUSSION TITLE
non-homepage routes give server errors on reload

### DISCUSSION THREAD

**Discussion by dave-edelhart-finfare** [vercel/community#4740] [Help] [answered]
### Page to Investigate

https://white-label-next.vercel.app/offers/b89a81

### Steps to Reproduce

any page but the home page gives server error on reload. This is a "server issues" as my local build does not exhibit this problem. The page loads fine when you click on an ad but when you reload the page (or go back from a jumpoff) you get a 500 server error. There are no errors in the deployment logs.

**Comment by dave-edelhart-finfare** [comment]
for the record I used vercel redirects to "patch" this but I do want to know why it doesn't work out of the box?

**Comment by amyegan** [comment, marked as answer]
I don't get a 500 error on the example page you shared. But I do see `RxDB dev-mode warning` and `Uncaught Error: Minified React error #418; visit https://legacy.reactjs.org/docs/error-decoder.html/?invariant=418 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.` in my browser console.

The [React 418 error decoder page](https://legacy.reactjs.org/docs/error-decoder.html/?invariant=418) says 
> Hydration failed because the initial UI does not match what was rendered on the server.

If you can share a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), someone here may be able to recognize what's causing the error.
